docker/ci/dockerfiles/current/build.centos7.opensearch.x64.arm64.dockerfile

```diff
@@ -97,6 +97,10 @@ RUN pip3 install pip==21.3.1
 RUN pip3 install cmake==3.21.3
 RUN pip3 install awscli==1.22.12
 
+# Install osslsigncode tool
+RUN arch=`uname -m` && yum clean all && curl -SLO https://download-ib01.fedoraproject.org/pub/epel/8/Everything/$arch/Packages/o/osslsigncode-2.2-1.el8.$arch.rpm && \
```

Comment:
This is not necessarily right as centos7 should not use rhel8 rpms.

Reply:
What do you suggest?

Reply:
I did not find the package for centos 7 https://download-ib01.fedoraproject.org/pub/centos/7/

Reply:
You are installing a rpm compiled for EL8 on EL7.
Though it might seems fine it is not officially supported.
Ex: https://access.redhat.com/articles/54483

Since we are doing the production here I recommend just do the compilation directly in dockerfile.
Or just get an additional image on Ubuntu2004, just for verify the signatures as it supports Ubuntu official repo.

Thanks.

Reply:
I reverted the change to build docker file. However for release clients I kept it for now. I tried compiling using the instructions here https://github.com/mtrojnar/osslsigncode#building but there were multiple build failures. We can use this image to sign and promote as well.